### PR TITLE
Removes metrics code

### PR DESCRIPTION
**Pull request by SimonRichardson**
We no longer support metrics code, so we should remove the code. Moving it to dqlite would require understanding and maintaining code that has not full tested or integrated with romulus or omnibus is sometime. Fully understanding if all parts are correct would be challenging.

If in the future we want to reuse parts, we can always look back in the history of the repo to bring back the parts that are required (if any).

### CODE REVIEW

**Review by Aflynn50 — COMMENTED** [review]
Checked out code and had a look for any remaining metrics/metering code, couldn't find any, looks good. I've approve but someone else with more context for this should review as well.

**Review by Aflynn50 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -55,11 +55,6 @@ All the other fields are optional.
 The type can be either a `string`, `int`, `float` or `boolean`. Everything else
 will cause Juju to error out when reading the charm.
 
-### `metrics.yaml`
-
-`metrics.yaml` represents an optional metrics gathering configuration yaml. For
-more information about metrics, read up on [Metric collecting charms](https://discourse.juju.is/t/metric-collecting-charms/1125)
```
Comment (hmlanigan) [inline README.md:61]: We need to find where else in documentation this needs to be removed.

**Review by hmlanigan — APPROVED** [review]
LGTM

**Pull request merged**